src/azul/indexer/document.py

```diff
@@ -1182,14 +1187,15 @@ def translate_fields(cls,
                             raise KeyError(f'Key {key!r} not defined in field_types')
                         except TypeError:
                             raise TypeError(f'Key {key!r} not defined in field_types')
-                        new_doc[key] = cls.translate_fields(val, field_type, forward=forward, path=(*path, key))
+                        new_doc[key] = cls.translate_fields(val, field_type, forward=forward, globs=globs,
+                                                            path=(*path, key))
```

Comment:
[CONTRIBUTING.rst 1.5 Line wrapping and indentation](https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#15line-wrapping-and-indentation), wrap all elements or none.
```suggestion
                        new_doc[key] = cls.translate_fields(val, 
                                                            field_type, 
                                                            forward=forward, 
                                                            globs=globs, 
                                                            path=(*path, key))
```